src/OpenColorIO/Config.cpp

```diff
@@ -1807,17 +1807,23 @@ void Config::validate() const
                 const char * csname = LookupRole(getImpl()->m_roles, name);
 
                 std::ostringstream os;
-                os << "Config failed validation. ";
+                os << "Config failed transform validation. ";
                 os << "This config references a color space, '";
 
                 if (!csname || !*csname)
                 {
-                    os << name << "', which is not defined.";
-                    getImpl()->m_validationtext = os.str();
-                    throw Exception(getImpl()->m_validationtext.c_str());
+                    // It's not a role, check to see if it's a named transform.
+                    if (!getImpl()->getNamedTransform(name.c_str()))
+                    {
+                        // It's not a color space, a role, or a named transform.
+                        os << name << "', which is not defined.";
+                        getImpl()->m_validationtext = os.str();
+                        throw Exception(getImpl()->m_validationtext.c_str());
+                    }
                 }
                 else if(!getImpl()->hasColorSpace(csname))
                 {
+                    // It's a role, but the color space it points to doesn't exist.
```

Comment:
Question: is a role allowed to point to a NamedTransform? In which case we would need to apply the same fix?

Reply:
Good question, but no, a named transform is not allowed to be a role (see line 1469 in validate).